recorder/sandbox.rb

```diff
@@ -46,6 +46,16 @@
   File.read(File.join('..', 'extension/fonts/glyphicons-halflings-regular.woff'))
 end
 
+get "/extension/fonts/fontawesome-webfont.ttf?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.ttf'))
+end
+
+get "/extension/fonts/fontawesome-webfont.woff?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.woff'))
+end
```

Comment:
What is going on here?  should we just make a custom font?  Could be small and clean and we'd only be concerned with the one file type.